src/modules/local_position_estimator/BlockLocalPositionEstimator.cpp

```diff
@@ -426,29 +515,31 @@ void BlockLocalPositionEstimator::updateHome()
 	_altHome = alt;
 	_gpsAltHome += delta_alt;
 	_baroAltHome +=  delta_alt;
-	_lidarAltHome +=  delta_alt;
-	_sonarAltHome +=  delta_alt;
+	_visionHome(2) += delta_alt;
```

Comment:
No, it is not. Currently it just assumes no offset, which works pretty well with my setup. If you had sonar and lidar both running and they were both offset at different heights, it would lead to some pretty bad oscillations, but that is an extremely unlikley edge case for most users. If we just add a few BlockParamFloats for lidarOffsetZ and sonarOffsetZ, it should be good for all use cases.